CalibForward/CTPPSPixelCalibration/BuildFile.xml

```diff
@@ -0,0 +1,9 @@
+<use   name="CondFormats/CTPPSReadoutObjects"/>
```

Comment:
hi @malbouis - it looks like the right subsystem for this package is CondTools. (eg, CondTools/CTPPS to keep things general)